src/app/ngrx/reducers/episode-metrics.reducer.ts

```diff
@@ -1,68 +1,83 @@
 import * as ACTIONS from '../actions';
 
 export interface EpisodeMetricsModel {
-  seriesId: number;
-  id: number;
+  podcastId: string;
   guid?: string;
   page?: number;
   monthlyReach?: any[][];
   weeklyReach?: any[][];
   dailyReach?: any[][];
   hourlyReach?: any[][];
+  charted?: boolean;
```

Comment:
whether this episode is charted is now kept here for in app state. that state charted resets when the podcast or page changes similar to what we're doing with the existing routing. but if the user just flips through the different chart display types (multi line or stacked, the charted episode state is retained)